plugins/hazelcast39/pom.xml

```diff
@@ -13,7 +13,7 @@
    <name>Hazelcast 3.9.x plugin for RadarGun</name>
 
    <properties>
-      <version.hazelcast>3.9.2</version.hazelcast>
+      <version.hazelcast>3.9</version.hazelcast>
```

Comment:
Why a less specific version? In https://github.com/radargun/radargun/pull/578 I set this to `3.9.4`, and made another change to depend on `plugin-hazelcast38` instead of `plugin-hazelcast37`.